src/graphql/HolofuelCounterpartyQuery.gql

```diff
@@ -0,0 +1,6 @@
+query HolofuelCounterparty($agentId: String!) {
+  holofuelCounterparty(agentId: $agentId) {
+    pubkey
```

Comment:
even if we're using `pubkey`, we should always be querying `id` as well, as that's what Apollo caching will use for identity.

Reply:
So the ID was explicitly removed and replaced with pubkey to prevent Apollo caching issues/conflicts arising for the need to (potentially) rewrite the HolofuelCounterparty ID returned from the new object with every new Query.  However, I agree that it isn't preferred to have a schema without an ID, as that creates an anomaly within our typing pattern.  I would be happy to pair and seek alternate solutions. :rocket:

Reply:
OK. I have restored the ID to this type and we can pursue any caching issues separately.